Sloth.Sql/dbo/Tables/Blobs.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE [dbo].[Blobs]
+(
+    [Id] NVARCHAR(450) NOT NULL PRIMARY KEY, 
```

Comment:
I prefer the primary key to be an int when possible.

Reply:
Really? A guid stored in nvarchar is all I've ever seen in just about every table of every CAES database I've looked at.

Reply:
I think the Sloth standard is the GUID ID so I think you did the right thing here.  For many years that was our standard but more recently we've been going with int IDs and I do think it's nicer.  But not for sloth.